daml/Marketplace/Notification.daml

```diff
@@ -0,0 +1,18 @@
+module Marketplace.Notification where
+
+import Marketplace.Utils
+
+
+template DismissibleNotification
+  with
+    sender : Party
+    senderRole : Optional MarketRole
+    receiver : Party
+    text : Text
+  where
+    signatory sender
+    observer receiver
```

Comment:
the `observer receiver` is redundant in this case. There is already a choice that the `receiver` party can exercise which means that DAML will automatically disclose this contract to the party. Doesn't hurt however having it there

Reply:
Makes sense, that's good to know.